shell_provisioner/module/sylius.sh

```diff
@@ -2,7 +2,7 @@
 
 cd /var/www/sites
 
-composer create-project --no-progress -s beta -n sylius/sylius-standard ./sylius 
+composer create-project --no-progress -s beta -n sylius/sylius-standard ./sylius
```

Comment:
Maybe we could extract comoser install/create to separate bash file and then we would have one source of truth for sylius installation, database setup and assets install

Reply:
Yes, seems to be nice.